desci-server/src/controllers/nodes/doi.ts

```diff
@@ -173,7 +175,7 @@ export const automateManuscriptDoi = async (req: RequestWithNode, res: Response,
     actions.push({ type: 'Update Title', title });
 
     // update contributors if populated
-    if (authors.length > 0) {
+    if (authors.length > 0 && !latestManifest.authors?.length) {
```

Comment:
Curious if this change would silently ignore the new entries in `authors` if `latestManifest.authors` already has some? I.e., prevent adding to an existing list? :eyes:  Maybe not a case that's supposed to happen, not sure.